src/NBitcoin/ChainedHeader.cs

```diff
@@ -36,6 +90,14 @@ public class ChainedHeader
         /// <summary>Total amount of work in the chain up to and including this block.</summary>
         public uint256 ChainWork { get { return Target.ToUInt256(this.chainWork); } }
 
+        public BlockDataAvailabilityState BlockDataAvailability { get; set; }
```

Comment:
Please add some documentation about what these are about.

Reply:
There is an open discussion about not needing to document every property as it will be repetitive.
I can copy past the comment from the enum itself but feels useless.

Reply:
@dangershony I think we can achieve an agreement in that if something is used in many components in the same way, we don't need that comment to be repeated.

This case is completely different and not documenting the property that is unique inside of the class would be a big mistake. This is not about repeating what each state value means, that information does not belong here.

Reply:
Okay, I understand, in which case I would suggest we introduce `<inheritdoc/>` here, or `<inheritdocfromtype/>`